rotkehlchen/chain/ethereum/modules/aave/aave.py

```diff
@@ -267,31 +266,31 @@ def _calculate_loss(
             if event.asset in earned_atoken_balances:
                 prev_balance = earned_atoken_balances[event.asset]
 
+            usd_price = query_usd_price_zero_if_error(
+                asset=event.asset,
+                time=event.get_timestamp_in_sec(),
+                location=f'aave interest event {event.event_identifier} from history',
+            )
+            event_balance = Balance(amount=event.amount, usd_value=event.amount * usd_price)
```

Comment:
here I cheated and created the balance manually since this is something that we want to deprecate and didn't want to spend time here